website/project/model.py

```diff
@@ -747,7 +747,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     wiki_private_uuids = fields.DictionaryField()
     file_guid_to_share_uuids = fields.DictionaryField()
 
-    creator = fields.ForeignField('user', backref='created')
+    creator = fields.ForeignField('user')
```

Comment:
It would be good to add an index to this field, since we will often query `Node` on this field alone (as you've implemented in `User.created`).